core/store/src/lib.rs

```diff
@@ -29,6 +33,7 @@ pub use crate::trie::{
 mod db;
 pub mod test_utils;
 mod trie;
+mod validate;
```

Comment:
let's move this to `store-validator` crate, we shouldn't need any private apis

Reply:
++